index.d.ts

```diff
@@ -0,0 +1,400 @@
+/// <reference types="node"/>
+import {ChildProcess} from 'child_process';
+import {Stream, Readable as ReadableStream} from 'stream';
+
+export type StdioOption =
+	| 'pipe'
+	| 'ipc'
+	| 'ignore'
+	| 'inherit'
+	| Stream
+	| number
+	| null
+	| undefined;
+
+export interface CommonOptions<EncodingType> {
+	/**
+	 * Current working directory of the child process.
+	 *
+	 * @default process.cwd()
+	 */
+	readonly cwd?: string;
+
+	/**
+	 * Environment key-value pairs. Extends automatically from `process.env`. Set `extendEnv` to `false` if you don't want this.
+	 *
+	 * @default process.env
+	 */
+	readonly env?: NodeJS.ProcessEnv;
```

Comment:
@BendingBender Do you know how we have to convince to reduce globals for the Node.js type definition and make stuff like this importable instead?

Reply:
@sindresorhus I suppose that making a PR on DefinitelyTyped is the easiest way to request feedback and start a discussion.

Just out of curiosity, is is just aesthetical or are there technical reasons why you want to migrate away from global usage?

Reply:
I don't want to put any work into it before I know whether it's an acceptable change. I guess I'll just open an issue about it.

> Just out of curiosity, is it just aesthetical or are there technical reasons why you want to migrate away from global usage?

I hate globals :P Node.js is all about explicitly importing things instead of dependending on globals (like the browser). Everything on `NodeJS` could just be imported instead.